packages/vscode/.vscodeignore

```diff
@@ -0,0 +1,32 @@
+# Ignore all files and directories in the parent directory
+../../
+../
+
+# Ignore some specific files and directories in the current directory
+.vscode/
+node_modules/
+imgs/
+.turbo/
+src/
+tsconfig.json
+packages.json
+
+# Ignore some specific files and directories in the parent directory
+.git
+.editorconfig
+.eslintrc.js
+.github/
+.gitignore
+.husky/
+.npmrc
+.travis.yml
+CONTRIBUTING.md
+Jenkinsfile.old
+LICENSE
```

Comment:
Should we allow LICENSE to be bundled in the vsix? With it ignored, I received the following prompt when running `npm run package`:
```
[WARNING] LICENSE.md, LICENSE.txt or LICENSE not found
Do you want to continue? [y/N]
```

Reply:
Good catch on this! I think it got removed when I re-worked the `.vscodeignore` on a previous commit